manifest.yml

```diff
@@ -48,9 +48,6 @@ DISTGEN_MULTI_RULES:
 
 # Files to copy
 COPY_RULES:
-  - src: src/content_sets.yml
```

Comment:
I do not really understand why have you removed this file from rules. Could you please explain that?

Reply:
Because file `content_sets.yml` is not valid for upstream. It is used only in dist-git and we do not synchronize it.